src/rust/src/orchard_ffi.rs

```diff
@@ -65,3 +73,108 @@ pub extern "C" fn orchard_bundle_serialize(
         }
     }
 }
+
+/// A signature within an authorized Orchard bundle.
+#[derive(Debug)]
+struct BundleSignature {
+    /// The signature item for validation.
+    signature: redpallas::batch::Item<SpendAuth, Binding>,
```

Comment:
If this API were not being used in per-transaction or per-block contexts, I would be concerned about unbounded memory usage here.

Reply:
Yeah, if we end up wanting to use this in a cross-blocks context then on queue pushes we could check the size of the batch, and run the batch in-line if it's above some threshold. That would require the queue push logic to be fallible though, so it might actually be better to at that point expose the batch size to the caller so it can decide when to trigger batch validation.